### PR TITLE
enterprise cloud adoption - redux

### PR DESCRIPTION
**Pull request by Petertaylor9999**
@bennage

### CODE REVIEW

**Comment by bennage** [comment]
Work remaining:
- [ ] cleaning up empty nodes and unused files
- [ ] redirections
- [ ] exposing appendix files in TOC (?)

**Comment by Petertaylor9999** [comment]
@bennage - with updated landing page and new overview in infra bucket

File: docs/toc.yml [hunk 1]
```diff
@@ -472,62 +472,38 @@
...
-            href: /azure/architecture/reference-architectures/virtual-machines-linux/single-vm
-            maintainContext: true
-  - name: Intermediate Azure adoption
+    - name: Overview
+      href: cloud-adoption/getting-started/overview.md
+    - name: How does Azure work?
+      href: cloud-adoption/getting-started/what-is-azure.md
+    - name: What is cloud resource governance?
+      href: cloud-adoption/getting-started/what-is-governance.md
+    - name: Resource access governance in Azure
```
Comment (MikeWasson) [inline docs/toc.yml]: In the title and links to this topic, it's called Resource access *management* in Azure

File: docs/cloud-adoption/governance/governance-single-team.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 
-The audience for this design guide is the *central IT* persona in your organization. *Central IT* is responsible for designing and implementing your organization's cloud governance architecture. As you learned in the [what is cloud resource governance?](governance-explainer.md) explainer, governance refers to the ongoing process of managing, monitoring, and auditing the use of Azure resources to meet the goals and requirements of your organization.
-
-The goal of this guidance is to help you learn the process of designing your organization's governance architecture by looking at a set of hypothetical governance goals and requirements. Then, we'll discuss how to configure Azure's governance tools to meet them. 
+The goal of this guidance is to help you learn the process for designing a resource governance model in Azure to support a single team and a simple workload.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
 
 In the foundational adoption stage, our goal is to deploy a simple workload to Azure. This results in the following requirements:
```
Comment (MikeWasson) [inline docs/cloud-adoption/governance/governance-single-team.md]: Are we still using the "adoption stage" model? With the restructuring, this is the first time it's been introduced (I think)

File: docs/cloud-adoption/governance/governance-single-team.md [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
+author: petertaylor9999
 ---
 
-# Azure governance design guide
+# Enterprise Cloud Adoption: Governance design for a simple workload
 
-The audience for this design guide is the *central IT* persona in your organization. *Central IT* is responsible for designing and implementing your organization's cloud governance architecture. As you learned in the [what is cloud resource governance?](governance-explainer.md) explainer, governance refers to the ongoing process of managing, monitoring, and auditing the use of Azure resources to meet the goals and requirements of your organization.
-
-The goal of this guidance is to help you learn the process of designing your organization's governance architecture by looking at a set of hypothetical governance goals and requirements. Then, we'll discuss how to configure Azure's governance tools to meet them. 
+The goal of this guidance is to help you learn the process for designing a resource governance model in Azure to support a single team and a simple workload.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements.
```
Comment (MikeWasson) [inline docs/cloud-adoption/governance/governance-single-team.md]: > several implementations

I think there's only one shown?

File: docs/cloud-adoption/governance/governance-single-team.md [hunk 4]
```diff
@@ -86,7 +84,19 @@ The lowest level of management scope is at the **resource** level. Operations ap
...
+
+When your Azure account is created, you specify a person in your organization to be the Azure **account owner**. An Azure Active Directory (Azure AD) tenant is then created by default. Your Azure **account owner** must [create the user account](/azure/active-directory/add-users-azure-active-directory) for the person in your organization who is the **workload owner**. 
+
+Next, your Azure **account owner** must [create a subscription](https://docs.microsoft.com/partner-center/create-a-new-subscription) and [associate the Azure AD tenant](/azure/active-directory/fundamentals/active-directory-how-subscriptions-associated-directory) with it.
+
+Finally, now that the subscription is created and your Azure AD tenant is associated with it, you can [add the **workload owner** to the subscription with the built-in **owner** role](/azure/billing/billing-add-change-azure-subscription-administrator#add-an-rbac-owner-for-a-subscription-in-azure-portal).
 
 ## Summary
```
Comment (MikeWasson) [inline docs/cloud-adoption/governance/governance-single-team.md]: Maybe add a sentence in the summary to the effect of: This is a simple governance model, with only one workload owner, but it's the basis for governing more complex teams/workloads?

File: docs/cloud-adoption/governance/governance-multiple-teams.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
+author: petertaylor9999
 ---
 
-# Governance design guide: new development in Azure for multiple teams
+# Enterprise Cloud Adoption: Governance design for multiple teams
 
-The goal of this guidance is to help you learn the process for designing a resource governance model in Azure.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
+The goal of this guidance is to help you learn the process for designing a resource governance model in Azure to support multiple teams, multiple workloads, and multiple environments.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
 
 Our requirements are:
```
Comment (MikeWasson) [inline docs/cloud-adoption/governance/governance-multiple-teams.md]: Using "our" here feels odd to me. Could say "The requirements" and "The individual in *your* organization"?

File: docs/cloud-adoption/governance/governance-multiple-teams.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
 * Identity management for multiple teams with different resource access requirements in Azure. The identity management system stores the identity of the following users:
```
Comment (MikeWasson) [inline docs/cloud-adoption/governance/governance-multiple-teams.md]: This confused me a little bit as to where these users came from. Maybe rephrase the requirement to something like "the organization plans to assign several individuals to different roles and therefore needs identity management...etc" (ie pivot to focus on the organizational need)

Comment (MikeWasson) [inline docs/cloud-adoption/governance/governance-multiple-teams.md]: Below,

> As you learned earlier, an environment is a logical grouping of resources

Looking through the earlier topics I don't see a definition of "environment" - maybe just define it here (which you do anyway)

Comment (MikeWasson) [inline docs/cloud-adoption/governance/governance-multiple-teams.md]: Should Figure 1 show creating a subscription? Otherwise it doesn't seem to relate to the sentence just above. (I think you could delete that sentence and figure, because you already state the account owner can create/delete subscriptions.)

Comment (MikeWasson) [reply docs/cloud-adoption/governance/governance-multiple-teams.md]: > No custom RBAC roles

What motivates this requirement? The other requirements meet obvious organizational needs, but this one isn't clear to me.

Comment (MikeWasson) [inline docs/cloud-adoption/governance/governance-multiple-teams.md]: In the section "Resource management model," some subheadings would be useful (H3's?)  because there's a lot of moving parts to keep in your head.

Maybe add a subhead here:

> Let's begin by evaluating the first option.

And another for the 2d option

File: docs/cloud-adoption/infrastructure/basic-workload.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+Before we begin, let's take a look at what the term workload actually means. It's a term that is typically understood to define an arbitrary unit of functionality such as an application or service. We think about a workload in terms of the code artifacts that are deployed to a server, but also in terms of any other services that are necessary. This is a useful definition for an on-premises application or service but in the cloud we need to expand on it.
+
+In the cloud, a workload not only encompasses all the artifacts but also includes the cloud resources as well. We include cloud resources as part of our definition because of a concept known as infrastructure-as-code. As you learned in the "how does Azure work" explainer, resources in Azure are deployed by an orchestrator service. The orchestrator service exposes this functionality through a web API, and this web API can be called using several tools such as Powershell, the Azure command line interface (CLI), and the Azure portal. This means that we can specify our resources in a machine-readable file that can be stored along with the code artifacts associated with our application.
```
Comment (MikeWasson) [inline docs/cloud-adoption/infrastructure/basic-workload.md]: > how does Azure work

link

File: docs/cloud-adoption/infrastructure/basic-workload.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+This isolation also enables another concept known as DevOps. DevOps includes the software development practices that include both software development and IT operations above, but adds the use of automation as much as possible. One of the principles of DevOps is known as continuous integration and continuous delivery (CI/CD). Continuous integration refers to the automated build processes that are run each time a developer commits a code change, and continuous delivery refers to the automated processes that deploy this code to various environments such as a development environment for testing or a production environment for final deployment.
+
+A basic web application or a virtual network (VNet) and virtual machine (VM). For more information about these different types of compute options in Azure, review the [overview of Azure compute options](/azure/architecture/guide/technology-choices/compute-overview?toc=/azure/architecture/cloud-adoption-guide/toc.json).
```
Comment (MikeWasson) [inline docs/cloud-adoption/infrastructure/basic-workload.md]: Missing something here?

File: docs/cloud-adoption/governance/governance-single-team.md (hunk 4 above)
Comment (Petertaylor9999) [reply docs/cloud-adoption/governance/governance-single-team.md]: Unfortunately it's really not though :/ 
I see what's missing though - we used to have "deploy a basic workload" as a next step, so I added that back in.

File: docs/cloud-adoption/governance/governance-multiple-teams.md (hunk 5 above)
Comment (Petertaylor9999) [reply docs/cloud-adoption/governance/governance-multiple-teams.md]: Yep, this is from when it was more progressive guidance and conversational in tone. This is fixed.

File: docs/cloud-adoption/governance/governance-multiple-teams.md (hunk 6 above)
Comment (Petertaylor9999) [reply docs/cloud-adoption/governance/governance-multiple-teams.md]: "Creating a subscription" is covered in the "Resource access management in Azure" doc. I'll link back to it.

Comment (Petertaylor9999) [reply docs/cloud-adoption/governance/governance-multiple-teams.md]: This whole section is going to be moved into the "getting started" section as its own restructured doc in the next iteration. So, let's leave this as is until then.

File: docs/cloud-adoption/infrastructure/basic-workload.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Enterprise cloud adoption: deploy a basic workload" 
+description: Describes how to deploy a basic workload to Azure
+author: petertaylor9999
+---
+
+# Enterprise cloud adoption: deploy a basic workload
+
+Before we begin, let's take a look at what the term workload actually means. It's a term that is typically understood to define an arbitrary unit of functionality such as an application or service. We think about a workload in terms of the code artifacts that are deployed to a server, but also in terms of any other services that are necessary. This is a useful definition for an on-premises application or service but in the cloud we need to expand on it.
```
[the remark this answers is not in the document]
Comment (Petertaylor9999) [reply docs/cloud-adoption/infrastructure/basic-workload.md]: This is an artifact of it being part of the other governance docs - I fixed it.

**Comment by Petertaylor9999** [comment]
@MikeWasson - I think I've addressed all your comments, and the merge conflict is resolved!
